integration-tests/js-compute/fixtures/app/src/out-of-memory.js

```diff
@@ -0,0 +1,18 @@
+import { pass, assert } from "./assertions.js";
+import { routes } from "./routes.js";
+import { sdkVersion } from "fastly:experimental";
+
+const starlingmonkey = sdkVersion.includes('starlingmonkey');
+
+routes.set("/oom", async () => {
+    if (starlingmonkey) {
+        return pass()
+    }
+    try {
+        const objects = [];
+        for (let i = 0; i < 1000; i++) {
+            objects.push(new BigUint64Array(1_000_000));
+        }
+    }
+    return fail('expected out of memory error')
```

Comment:
Does `fail` need to be imported?

Reply:
Thanks for catching this, this wasn't supposed to be committed, and comes from https://github.com/fastly/js-compute-runtime/pull/805, but wasn't landed as we don't currently have a good stdout testing mechanism.